src/app/tests/suites/TestGroupKeyManagementCluster.yaml

```diff
@@ -23,159 +23,209 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
+    - label: "Read maxGroupsPerFabric"
+      command: "readAttribute"
+      attribute: "maxGroupsPerFabric"
+      response:
+          value: 2
```

Comment:
This value may not be stable per device: you expect >= 2, not == 2.

Suggest to replace with:
```
constraints:
  minValue: 2
```

same for the other "fixed" responses here

Reply:
Fixed as suggested.